frontend/portals/Media/index.jsx

```diff
@@ -69,6 +74,8 @@ css.global('.tablet-right-column .price ui-shared__price', {
   fontSize: '1.7rem',
 });
 
+const NATIVE_SHARE_BUTTON = 'product.tablet.right-column.ctas';
```

Comment:
i would prefer a more common name for the const that matches also more the actual value of the const

Reply:
💯